ts/test/tec_test.ts

```diff
@@ -1124,13 +1132,21 @@ function onMessage(client: WebSocket.w3cwebsocket, messageNumber: number): Array
     return promises;
 } // tslint:disable:max-file-line-count
 
-function createSignedTransaction(
-    data: string,
+async function createSignedTransactionAsync(
+    transactionData: Partial<SignedZeroExTransaction>,
     signerAddress: string,
-    transactionData?: Partial<SignedZeroExTransaction>,
-): SignedZeroExTransaction {
+    chainId: number,
+): Promise<SignedZeroExTransaction> {
     const privateKey = TESTRPC_PRIVATE_KEYS[accounts.indexOf(signerAddress)];
-    transactionFactory = new TransactionFactory(privateKey, contractAddresses.exchange);
-    const signedTransaction = transactionFactory.newSignedTransaction(data, SignatureType.EIP712, transactionData);
+    transactionFactory = new TransactionFactory(privateKey, contractAddresses.exchange, chainId);
+
+    // if not provided, set default expiration time to be 90s in the future
+    const expirationTimeSeconds =
+        transactionData.expirationTimeSeconds || new BigNumber(Math.floor(Date.now() / 1000) + 90); // tslint:disable-line:no-magic-numbers
+    const txData = {
+        expirationTimeSeconds,
+        ...transactionData,
+    };
```

Comment:
nit:
```
const txData = {
    expirationTimeSeconds: new BigNumber(Math.floor(Date.now() / 1000) + 90), // tslint:disable-line:no-magic-numbers
    ...transactionData,
};
```

Reply:
Actually, the `90` is should always equal the Coordinator Server's `EXPIRATION_DURATION_SECONDS` config value. We could future proof this constant by pulling from the test config.

Reply:
oh cool, didnt realise the `Object.spread` function would take precedence!